src/eduid/common/models/bearer_token.py

```diff
@@ -0,0 +1,220 @@
+import logging
+from copy import copy
+from enum import Enum
+from typing import Any, Mapping, Optional
+
+from pydantic import BaseModel, Field, StrictInt, root_validator, validator
+
+from eduid.common.config.base import AuthnBearerTokenConfig, DataOwnerName, ScopeName
+from eduid.userdb.scimapi.groupdb import ScimApiGroupDB
+
+
+class AuthSource(str, Enum):
+    INTERACTION = "interaction"
+    CONFIG = "config"
+    MDQ = "mdq"
+    TLSFED = "tlsfed"
+
+
+class SudoAccess(BaseModel):
+    type: str
+    scope: ScopeName
```

Comment:
tror att scope måste bli optional i den här och scimapi måste lära sig hantera det, annars kan man inte bara skicka in type som vi vill göra till maccapi. sen kan vi fundera på om SudoAccess är det rätta namnet längre men det behöver inte ändras nu.

Reply:
Såg inte att scimapi hade specifik kod utan det jag hittade låg i AuthnBearerToken